### PR TITLE
Nutrition table 99

### LINKED ISSUE #99: Display full nutrition table
Something like: calcium, iron, proteins (what amino acids), vitamins (A, C, …), omega (3, 6,…), etc

### PR DESCRIPTION
**Pull request by herau**
According to Europe disposition 
https://github.com/openfoodfacts/openfoodfacts-server/blob/039d2a092d2b8c54f2c1fc03be41a94ae02d40ed/lib/ProductOpener/Food.pm#L212

fix #99

refactor the `Nutriments` class to get all the nutriments (instead of binding one by one in a java class field)

screenshot_1485437541
screenshot_1485437567
screenshot_1485437558

### CODE REVIEW

**Comment by herau** [comment]
fix #99

**Comment by teolemon** [comment]
Well done 🥇

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/views/adapters/NutrimentsRecyclerViewAdapter.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    private final List<NutrimentItem> nutrimentItems;
+
+    public NutrimentsRecyclerViewAdapter(List<NutrimentItem> nutrimentItems) {
+        super();
+        this.nutrimentItems = nutrimentItems;
+    }
+
+    @Override
+    public RecyclerView.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        if (viewType == TYPE_ITEM) {
```
Comment (itchix) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/views/adapters/NutrimentsRecyclerViewAdapter.java]: You can also write it like that : https://github.com/openfoodfacts/OpenFoodFacts-androidApp/blob/master/app/src/main/java/openfoodfacts/github/scrachx/openfood/views/adapters/ProductsRecyclerViewAdapter.java#L41 
It's only to be homogeneous, but I'm fine with that 👍

Comment (teolemon) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/views/adapters/NutrimentsRecyclerViewAdapter.java]: Creating an issue to remind us to fix this. Let's merge.

**Comment by teolemon** [comment]
Merged. Well done @herau

**Pull request merged**